metrics/metrics.go

```diff
@@ -0,0 +1,115 @@
+package metrics
+
+import (
+	"context"
+	"net"
+	"time"
+
+	"github.com/lucas-clemente/quic-go/logging"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+// Measures
+var (
+	connections = stats.Int64("connections", "number of QUIC connections", stats.UnitDimensionless)
```

Comment:
Should we have a quic prefix here, so this doesn't cause conflicts? Something like `/quic/connections`? (Not sure what the standard way of doing this is with opencensus.)

Reply:
I went with `/quic-go/connections` for now. Happy to change this if @lanzafame has another suggestion.